sw/airborne/modules/mav_course_exercise/mav_exercise.c

```diff
@@ -0,0 +1,166 @@
+//
+// Created by matteo on 31/05/2020.
+//
```

Comment:
(minor: please copy the full header from one of the other modules, also in .h)

Reply:
Some modules have a slightly different header, I wasn't sure what the latest one was (or if there's a difference), but I changed it

Reply:
Some modules have a slightly different header, I wasn't sure what the latest one was (or if there's a difference), but I changed it

Reply:
:+1: